spec/fortnox/api/validators/invoice_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'spec_helper'
+require 'fortnox/api/models/invoice'
+require 'fortnox/api/validators/context'
+require 'fortnox/api/validators/invoice'
+
+describe Fortnox::API::Validator::Invoice do
+  let( :model_class ){ Fortnox::API::Model::Invoice }
+  let( :valid_model ){ model_class.new( customer_number: 12345 ) }
```

Comment:
Here you could argue it's ok (though it's a constant and constant lookup is pretty optimal as is in Ruby). The other problem with it is that you are basically creating a second pointer to a class and classes (all constants really) should begin with a capital letter. So if you want to do it you should really call it `Model` instead :)